tests/console/login.go

```diff
@@ -77,7 +77,22 @@ func LoginToAlpine(vmi *v1.VirtualMachineInstance) error {
 	}
 	defer expecter.Close()
 
+	err = expecter.Send("\n")
```

Comment:
Why lines 80-84 are needed? isn't the batcher enough?

Reply:
Its basically enough, but those are little faster, sanity that we can even send,
saw it as well on fedora and Cirros so added here